README.md

```diff
@@ -172,6 +172,52 @@ To check, go to  PyCharm | Settings | Languages & Frameworks | Django and check
 
 Now that you are done, you can run Django server using PyCharm and access the studio at [http://studio.127.0.0.1.nip.io/](http://studio.127.0.0.1.nip.io/)
 
+## Testing
+
+The automated tests in this project consist of unit tests, end2end UI tests and integration tests.
+
+### Unit tests
+
+The unit tests are written in Python and located in several /tests/ directories under each applicable part of the codebase. The tests are automatically executed as part of git workflows. You may also run them manually in this manner using docker compose:
+
+```bash
+docker compose up -d --build
+docker compose run unit-tests
+```
+
+### End2End tests
+
+UI end2end tests and Integration tests use the [Cypress](https://www.cypress.io/) framework and are implemented in javascript. The UI end2end are automatically executed as part of git workflows. You may also run them manually in this manner after first installing Cypress on your machine:
```

Comment:
I think it's worth mentioning that this requires a running instance of serve on a local machine.